ru/Алгоритмы Сортировки/Флаг Нидерландов.md

```diff
@@ -0,0 +1,39 @@
+# Задача флага Нидерландов
+
+#### Постановка задачи
+
+Дан массив с несколькими цветами (красный, белый и синий), нужно упорядочить его таким образом, чтобы все цвета были сгруппированы вместе. Мы будем использовать целые числа `0`, `1` и `2` для представления красного, белого и синего цветов соответственно. 
+
+Нужно отсортировать массив на месте без использования дополнительной памяти и стандартной функции сортировки.
+
+#### Алгоритм
+
+1. Создайте три указателя - `low`, `mid` и `high`.
+1. Установите `low` и m`id в начало массива, а `high` в конец массива.
```

Comment:
it looks like you forgot to close a `monospace` here?

Reply:
fixed, thank you